scrooge-generator/src/main/resources/scalagen/finagleService.mustache

```diff
@@ -115,7 +115,7 @@ class {{ServiceName}}$FinagleService(
       val service = serviceMap.get(msg.name)
       service match {
         case _root_.scala.Some(svc) =>
-          svc(iprot, msg.seqid)
+          svc((iprot, msg.seqid))
```

Comment:
Second thought, I believe you need to also make this change to GoldService$FinagleService.scala and PlatinumService$FinagleService.scala. Can you run the test to confirm?

Reply:
Only needed to add to GoldService :smile: